src/App.js

```diff
@@ -55,6 +63,14 @@ class App extends Component {
 
     return candidates[Math.floor(Math.random() * candidates.length)];
   }
+
+  getLastFiveSelectedValues(lastFiveSelectedValues, nextSelectedValue) {
+    lastFiveSelectedValues.push(nextSelectedValue);
+    if (lastFiveSelectedValues.length > 5) {
+      lastFiveSelectedValues.shift();
```

Comment:
So another approach here could be to not shift here, and store the entire history in the state, and then trim to the last 5 values just in the UI. No strong feelings about which one is better, though.